source/options.js

```diff
@@ -18,8 +16,11 @@ function updateInputField(event) {
 }
 
 for (const input of rangeInputs) {
-	input.addEventListener('input', updateColor);
+	input.addEventListener('input', updateOutputColor);
 	input.addEventListener('input', updateInputField);
 }
 
-window.addEventListener('load', updateColor);
+window.addEventListener('load', async () => {
```

Comment:
Maybe I should use a function declaration instead to be consistent with the rest of this file?

```js
async function onLoad() {
	await optionsStorage.syncForm('#options-form');
	updateOutputColor();
}
// ...
window.addEventListener('load', onLoad);